packages/lab/src/overlay/OverlayPanelBase.tsx

```diff
@@ -0,0 +1,55 @@
+import { FloatingArrow } from "@floating-ui/react";
+
+import overlayCss from "./Overlay.css";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { Button, makePrefixer } from "@salt-ds/core";
+import { CloseIcon } from "@salt-ds/icons";
+import { ComponentPropsWithoutRef } from "react";
+import { useOverlayContext } from "./OverlayContext";
+import clsx from "clsx";
+
+const withBaseName = makePrefixer("saltOverlay");
+
+interface OverlayPanelBaseProps extends ComponentPropsWithoutRef<"div"> {}
+
+export const OverlayPanelBase = (props: OverlayPanelBaseProps) => {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-overlay",
+    css: overlayCss,
+    window: targetWindow,
+  });
+
+  const { arrowProps, handleCloseButton } = useOverlayContext();
+  const { children, className, ...rest } = props;
+
+  return (
```

Comment:
do we need to have overlayPanel and overlayPanelBase? most old components that had a base in have been merged, could the same not happen in overlay?